plugin.video.nba/resources/language/English/strings.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
In next update please make use of strings.po, rather than strings.xml
Please see http://kodi.wiki/view/Language_support#What_is_strings.po